src/python/turicreate/_deps/__init__.py

```diff
@@ -38,6 +42,13 @@ def __get_version(version):
     from . import pandas_mock as pandas
 
 
+if not HAS_PANDAS and is_py_interactive() and is_minimal_pkg():
+    print(
```

Comment:
We shouldn't be printing out this message. We should be handling panda imports the same way we're handling TensorFlow imports.

Reply:
The way we handle pandas is different. For other packages, we use lazy-import. But for `pandas`, we use feature flags. Once it's turned off by `HAS_PANDAS`, it will be turned off everywhere. So let the user know if they don't have pandas, they won't have pandas functionality support.

However, remove this is also reasonable, if they don't have pandas, why they should use pandas?

I will remove this prompt.